docs/2.10.0/docs/canton/usermanual/security.rst

```diff
@@ -320,6 +320,29 @@ A fingerprint of a key can be retrieved from the list of public keys:
    :end-before: user-manual-entry-end: ListPublicKeys
    :dedent:
 
+.. _deleting-canton-keys:
+
+Deleting Canton Node Keys
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Deletion of keys, in particular, private keys is not done automatically in Canton. When a key is rotated, the previous
+key remains stored, but is disabled and no longer used. To permanently delete a key, use the following Canton console
+command:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/scala/com/digitalasset/canton/integration/tests/security/topology/KeyManagementIntegrationTest.scala
+   :language: scala
+   :start-after: user-manual-entry-begin: DeletePrivateKey
+   :end-before: user-manual-entry-end: DeletePrivateKey
+   :dedent:
+
+.. warning::
+  Exercise caution when deleting a private key. You must ensure that the key is no longer in use and is not needed
+  for decrypting or verifying signatures of previous messages. Therefore, keys must only be deleted after you have
```

Comment:
A private key is not needed to verify signatures. You need the corresponding public key for verification and this section is about the private key.